web-portal/frontend/Dockerfile

```diff
@@ -8,10 +8,8 @@ COPY . .
 ENV NEXT_TELEMETRY_DISABLED 1
 
 # INJECT ENV VARIABLES AT BUILD TIME
-ARG NEXT_PUBLIC_API_ENDPOINT
-ARG NEXT_PUBLIC_RPC_ENDPOINT
-ARG NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID
-ARG NEXT_PUBLIC_APP_URL
+ARG API_ENDPOINT
```

Comment:
I still think we should figure out how to avoid this, but its ok for now. We should document it clearly though, since it isn't obvious why this is done this way.

Reply:
This is understandable needs a comment/flag